extensions/hibernate-orm/deployment/src/main/java/io/quarkus/hibernate/orm/deployment/HibernateOrmConfigPersistenceUnit.java

```diff
@@ -381,4 +396,32 @@ public boolean isAnyPropertySet() {
         @ConfigItem
         public OptionalLong objectCount;
     }
+
+    @ConfigGroup
+    public static class HibernateOrmConfigPersistenceUnitFetch {
+        /**
+         * The size of the batches used when loading entities and collections.
+         *
+         * `-1` means batch loading is disabled. This is the default.
+         *
+         * @asciidoclet
+         */
+        @ConfigItem(defaultValue = "-1")
+        public int batchFetchSize;
```

Comment:
Does this make the property name `.fetch.batch-fetch-size`? Can we eliminate the redundancy and call it `.fetch.batch-size`?

Reply:
Yes, new properties' names would be `.fetch.batch-fetch-size` and `.fetch.max-fetch-depth`. I am okay with removing the `fetch` keyword as far as redundancy is concerned. Shall we also do the same for `maxFetchDepth` so that it is `.fetch.max-depth` ?

Reply:
Right, I mean for both of them.

Reply:
@mkrzywanski can you adjust the names and squash everything?

Thanks!

Reply:
@gsmet done.